roles/cockpit/tasks/main.yml

```diff
@@ -6,10 +6,11 @@
     - cockpit-shell
     - cockpit-bridge
     - "{{ cockpit_plugins }}"
-  when: not openshift.common.is_atomic | bool
+  when: not openshift.common.is_containerized | bool
 
 - name: Enable cockpit-ws
   service:
     name: cockpit.socket
     enabled: true
     state: started
+  when: not openshift.common.is_containerized | bool
```

Comment:
I just don't want to do anything with cockpit for containerized installs until there actually is a containerized cockpit.